src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -38,6 +38,10 @@ library GPv2Encoding {
     /// @dev The stride of an encoded order.
     uint256 internal constant ORDER_STRIDE = 204;
 
+    /// @dev The order EIP-712 type hash.
```

Comment:
Please add to comment what keccak computation is used to derive at this constant.